tests/test_cellranger.py

```diff
@@ -0,0 +1,77 @@
+import os
+import pytest
+import shutil
+
+from tests.util import s3, hash
+import toolchest_client as toolchest
+
+toolchest_api_key = os.environ.get("TOOLCHEST_API_KEY")
+if toolchest_api_key:
+    toolchest.set_key(toolchest_api_key)
+
+MIN_EXPECTED_ARCHIVE_SIZE = 34000000
+MAX_EXPECTED_ARCHIVE_SIZE = 38000000
+
+EXPECTED_SUMMARY_SIZE = 2744825
+EXPECTED_RAW_MATRIX_SIZE = 868393
+EXPECTED_RAW_MATRIX_HASH = "d00cca1d2b4344b03946eeaeedc17ed5"
+EXPECTED_FILTERED_MATRIX_SIZE = 503956
```

Comment:
Is there a benefit to abstracting these as global constants rather than just having them locally in the assertion?

Reply:
This is the format that we've been following for the other tests, but we can move them locally into `verify_cellranger_count_outputs()`.

Reply:
I started this pattern with Kraken 2 because it needed to be referenced twice, outside the scope of the same function (https://github.com/trytoolchest/toolchest-client-python/blob/main/tests/test_kraken2.py).

It's harder to reason about these things at a glance if they're abstracted / moved to the non-relevant part of the file.

Reply:
If it exists improperly in other tests, that's my fault and and I should have caught it earlier.

Reply:
Sounds good, moving to `verify_cellranger_count_outputs()`